source/module_esolver/esolver_ks_lcao.cpp

```diff
@@ -415,14 +415,9 @@ namespace ModuleESolver
                 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                 GlobalC::pot.vr = GlobalC::pot.v_of_rho(GlobalC::CHR.rho, GlobalC::CHR.rho_core);
                 GlobalC::en.delta_escf();
-                if (ELEC_evolve::td_vext == 0)
-                {
+                
```

Comment:
Please 1. keep the code aligned and 2. add some annotation here to explain what will next code do.